entity-types/infra-kubernetes_persistentvolumeclaim/definition.yml

```diff
@@ -41,4 +41,11 @@ synthesis:
           present: false
         # value added for test entities only
         - attribute: newrelicOnly
-          value: "true"
\ No newline at end of file
+          value: "true"
+      tags:
+        persistentvolumeclaim:
+          entityTagNames: [k8s.pvcName]
+        namespace:
+          entityTagNames: [k8s.pvcNamespace]
```

Comment:
Namespace tag is consistently named as `k8s.namespace` in all other entities, but here it contains the `pvc` prefix. Shouldn't we remove the prefix?

Reply:
It agree that `k8s.namespace` makes more sense – but `k8s.pvcNamespace` is the existing golden tag for `perisistentvolumeclaim` – see line 13.